### ISSUE TITLE
Leading zeros

### ISSUE THREAD

**Issue by amoibos** [circlecell/jsonlint.com#82]
If I you use leading zeros for numbers I'm get false negatives with wrong position and error message of the problem. In my case it was an array with objects which contains key value pairs where the value was a number.

**Comment by onelharrison** [comment]
@amoibos The definition of a number for does not allow leading zeros.

Take a look at this comment by @finom on a related [issue](https://github.com/circlecell/jsonlint-mod/pull/1#issuecomment-392321385).

I wasn't able to reproduce a misleading error message using data in the format you described (shown below).

```json
[
  { "key": 1},
  { "key": 01}
]
```
